site/source/cookbook/modern-web-apps/efficient-state-management.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: docs
+category: modern-web-apps
+title: Manage state in your application
+overview: Manage state to decrease complexity in your application
+---
+
+## Objective
+
+This recipe demonstrates a technique for managing state in your Dojo 2 application.
```

Comment:
Should we be specific, that it demonstrates "injecting" or "side loading" state into a widget in your application? There are actually multiple ways to manage state, this is just one example (that can be used in combination with the others)

Reply:
Added a note on this